src/libs/apis/GroupComplete/getGroupInfo.ts

```diff
@@ -0,0 +1,11 @@
+import { api } from '../../api';
+
+export const getGroupInfo = async (uuid: string) => {
+  try {
+    const response = await api.get(`rooms/uuid/${uuid}`);
+    const { data } = response.data;
```

Comment:
P5: 원래 하던 것처럼로 하면 한 줄의 코드를 줄일 수 있을 것 같아요!
```suggestion
    const { data } = await api.get(`rooms/uuid/${uuid}`);
```

Reply:
아하 수정하겠습니다 감사합니다 !